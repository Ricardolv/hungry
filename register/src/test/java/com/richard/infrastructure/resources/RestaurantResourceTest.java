package com.richard.infrastructure.resources;

import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;

@QuarkusTest
public class RestaurantResourceTest {

    @Test
    public void testHelloEndpoint() {
        given()
          .when().get("/records")
          .then()
             .statusCode(200)
             .body(is("Hello from RESTEasy Reactive"));
    }

}