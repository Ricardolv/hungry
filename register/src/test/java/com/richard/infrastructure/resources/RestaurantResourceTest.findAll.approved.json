[
    {
        "id": 123,
        "name": "Novo restaurante",
        "owner": "ID do Keycloak"
    },
    {
        "id": 456,
        "name": "Novo restaurante 2",
        "owner": "ID do Keycloak 2"
    }
]